packages/transformers/js/src/JSTransformer.js

```diff
@@ -392,6 +399,7 @@ export default (new Transformer({
       replace_env: !asset.env.isNode(),
       inline_fs: Boolean(config?.inlineFS) && !asset.env.isNode(),
       insert_node_globals: !asset.env.isNode(),
+      relative_context: Boolean(config?.relativeContext),
```

Comment:
`relative_context` was just an arbitrary name chosen by me to get things going.

I'll need input here on the proper name + shape of the API how it should look